src/matter/session/secure/CaseMessages.ts

```diff
@@ -15,56 +23,79 @@ export const KDFSR3_INFO = Buffer.from("Sigma3");
 export const TBE_DATA2_NONCE = Buffer.from("NCASE_Sigma2N");
 export const TBE_DATA3_NONCE = Buffer.from("NCASE_Sigma3N");
 
+/** @see {@link MatterCoreSpecificationV1_0} § 3.5.1 */
+const CRYPTO_GROUP_SIZE_BITS = 256;
+const CRYPTO_GROUP_SIZE_BYTES = 32;
+const CRYPTO_PUBLIC_KEY_SIZE_BYTES = (2 * CRYPTO_GROUP_SIZE_BYTES) + 1;
+
+/** @see {@link MatterCoreSpecificationV1_0} § 3.3 */
+const CRYPTO_HASH_LEN_BYTES = 32;
+const CRYPTO_HASH_BLOCK_LEN_BYTES = 64;
+
+/** @see {@link MatterCoreSpecificationV1_0} § 3.6 */
+const CRYPTO_SYMMETRIC_KEY_LENGTH_BITS = 128;
+const CRYPTO_SYMMETRIC_KEY_LENGTH_BYTES = 16;
+const CRYPTO_AEAD_MIC_LENGTH_BITS = 128;
+const CRYPTO_AEAD_MIC_LENGTH_BYTES = 16;
+const CRYPTO_AEAD_NONCE_LENGTH_BYTES = 13;
+
+/** @see {@link MatterCoreSpecificationV1_0} § 4.13.2.3 */
 export const CaseSigma1T = ObjectT({
-    random: Field(1, ByteStringT()),
-    sessionId: Field(2, UnsignedIntT),
-    destinationId: Field(3, ByteStringT()),
-    ecdhPublicKey: Field(4, ByteStringT()),
-    mrpParams: OptionalField(5, ObjectT({
+    random: Field(1, ByteStringT({ length: 32 })),
+    sessionId: Field(2, BoundedUnsignedIntT({ min: 0, max: 0xFFFF })),
```

Comment:
Ok, will try it tomorrow

Reply:
I will write a PR now to unblock this.

Reply:
As proposed in other PR: Leave old types and "derecate" them, lets merge stuff and I will do a run over codebase for that and provide PR. So it would be not blocking each other